crates/db_schema/src/impls/comment.rs

```diff
@@ -287,26 +286,27 @@ impl Likeable for CommentActions {
     pool: &mut DbPool<'_>,
     person_id: PersonId,
     comment_id: Self::IdType,
-  ) -> LemmyResult<uplete::Count> {
+  ) -> LemmyResult<usize> {
     let conn = &mut get_conn(pool).await?;
-    uplete::new(comment_actions::table.find((person_id, comment_id)))
-      .set_null(comment_actions::like_score)
-      .set_null(comment_actions::liked_at)
-      .get_result(conn)
+    diesel::update(comment_actions::table.find((person_id, comment_id)))
+      .set((
+        comment_actions::like_score.eq(None::<i16>),
+        comment_actions::liked_at.eq(None::<DateTime<Utc>>),
+      ))
```

Comment:
For some reason diesel makes you define the type as `None` when setting to null. Kind of annoying but nbd.